snippets/csharp/new-in-6/initializers.cs

```diff
@@ -5,11 +5,11 @@ namespace Initializers
     public class Examples
     {
         // <ListInitializer>
```

Comment:
This snippet tag says ListInitializer bu t it's using an IndexInitializer ?

Reply:
Changed here, and in the use in dotnet/docs#9551